src/plots/Volume/VolumeAttributes.code

```diff
@@ -651,6 +651,3 @@ VolumeAttributes_SetMaterialProperties(PyObject *self, PyObject *args)
     return Py_None;
 }
 
-Target: xml2cmake
```

Comment:
These lines should be put back in.
Not sure why you keep running into this problem.
I will create a ticket to checkout xmledit to see why these get lost.

Reply:
Yeah...I caught the last one and missed this. Good catch.